### PR TITLE
Modify virt-v2v functional case84637

### PR DESCRIPTION
**Pull request by mxie91**
As previous case only cover the sceanrio about rhel7 efi guest,
add a scenario about rhel6 efi guest.

Signed-off-by: mxie91 <mxie@redhat.com>

### CODE REVIEW

**Comment by mxie91** [comment]
# avocado run --vt-type v2v function_test_esx.positive_test.uefi.ovmf.rhel7.esx_67.libvirt.it_default
JOB ID     : 59914449dc64d2f4b379a2a4be05fd5ae9a109ec
JOB LOG    : /root/avocado/job-results/job-2019-07-19T03.44-5991444/job.log
 (1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.uefi.ovmf.rhel7.esx_67.libvirt.it_default: PASS (3755.17 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 3756.31 s

# avocado run --vt-type v2v function_test_esx.positive_test.uefi.ovmf.rhel6.esx_67.libvirt.it_default
JOB ID     : 2f68c8d92f44abbcabecbadef5a56522706d2173
JOB LOG    : /root/avocado/job-results/job-2019-07-24T23.02-2f68c8d/job.log
 (1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.uefi.ovmf.rhel6.esx_67.libvirt.it_default: PASS (1824.27 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 1825.25 s

**Comment by xiaodwan** [comment]
@chloerh I suddenly think of putting rhel6 case into blacklist is better than skipping the checking in script. what do you think?

File: v2v/tests/cfg/function_test_esx.cfg [hunk 1]
```diff
@@ -118,10 +118,15 @@
                             checkpoint = 'ovmf'
                             msg_content = 'virt-v2v: warning: fstrim on guest filesystem /dev/.*? failed.  Usually'
                             expect_msg = yes
-                        - rhel:
-                            only esx_55
-                            checkpoint = uefi_rhel
-                            main_vm = VM_NAME_ESX_UEFI_RHEL_V2V_EXAMPLE
+                        - rhel6:
+                            only esx_67
+                            main_vm = Auto-esx6.7-rhel6.10-efi
```
Comment (xiaodwan) [inline v2v/tests/cfg/function_test_esx.cfg]: You can't write a real name here.

**Comment by chloerh** [comment]
> @chloerh I suddenly think of putting rhel6 case into blacklist is better than skipping the checking in script. what do you think?

If we don't need to test them, maybe just remove them. But I'm ok with either way as long as it meets our test.

**Pull request closed without merging**